docs/docs/external-volumes

```diff
@@ -62,15 +62,21 @@ You specify an external volume in the app definition of your Marathon app. [Lear
 }
 ```
 
-In the `volumes` section of app definition above:
+In the app definition above:
+
+- `containerPath` specifies where the volume is mounted inside the container. This path is configurable, but defaults to `data/`. See [the REX-Ray documentation on data directories](https://rexray.readthedocs.org/en/v0.3.2/user-guide/config/#data-directories) for more information.
+
+- The `external.driver["dvdi/driver"]` option specifies which Docker volume driver to use for storage. If you are running Marathon on DCOS, this value should always be `rexray`. [Learn more about REX-Ray](https://rexray.readthedocs.org/en/v0.3.2/user-guide/schedulers/).
+
+- You can specify additional options with `container.volumes[x].external.options[optionName]`.
```

Comment:
the "dvdi" provider, for MESOS containers, uses "dvdcli" which documents
the following (rexray-focused) options:
https://github.com/emccode/dvdcli#extra-options. Marathon provides some
validation for some of these. We've also enabled a general "pass-through"
so that an option named "dvdi/optionABC" is passed through to the dvdcli as
"optionABC".

the "dvdi" provider, for DOCKER containers, only supports the "dvdi/driver"
option.

options are generally intended to target specific provider implementations.
the "dvdi" implementation is somewhat complex in that it supports multiple
drivers below it (and those drivers have arbitrary options).



On Wed, Apr 6, 2016 at 5:54 PM, Suzanne Scala <notifications@github.com>
wrote:

> In docs/docs/external-volumes
> <https://github.com/mesosphere/marathon/pull/3607#discussion_r58788958>:
>
> > @@ -62,15 +62,21 @@ You specify an external volume in the app definition of your Marathon app. [Lear
> >  }
> >  ```
> >
> > -In the `volumes` section of app definition above:
> > +In the app definition above:
> > +
> > +- `containerPath` specifies where the volume is mounted inside the container. This path is configurable, but defaults to `data/`. See [the REX-Ray documentation on data directories](https://rexray.readthedocs.org/en/v0.3.2/user-guide/config/#data-directories) for more information.
> > +
> > +- The `external.driver["dvdi/driver"]` option specifies which Docker volume driver to use for storage. If you are running Marathon on DCOS, this value should always be `rexray`. [Learn more about REX-Ray](https://rexray.readthedocs.org/en/v0.3.2/user-guide/schedulers/).
> > +
> > +- You can specify additional options with `container.volumes[x].external.options[optionName]`.
>
> @jdef <https://github.com/jdef> One final question: how can the user
> learn what other options are available?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/marathon/pull/3607/files/79d9d403e7d7a4363c431f53a5d70f2dad618f55..9f27a0eddf39a982ef50dd3dad462686e9db81d3#r58788958>
>